core/src/HIP/Kokkos_HIP_Abort.hpp

```diff
@@ -50,6 +50,18 @@
 
 #include <hip/hip_runtime.h>
 
+// FIXME_HIP ROCm 4.5 version header include would be <rocm/rocm_version.h>
+#if __has_include(<rocm_version.h>)
+#include <rocm_version.h>
+#define KOKKOS_IMPL_ROCM_VERSION \
+  ROCM_VERSION_MAJOR * 10000 + ROCM_VERSION_MINOR * 100 * ROCM_VERSION_PATCH
```

Comment:
This looks wrong. I don't remember exactly but the following makes more sense
```suggestion
  ROCM_VERSION_MAJOR * 10000 + ROCM_VERSION_MINOR * 100 + ROCM_VERSION_PATCH
```

Reply:
Yes it is a typo.  I haven't tested yet.

Reply:
OK I got around to test it now and it does what I want